doc/lua_api.txt

```diff
@@ -625,14 +648,24 @@ which it assumes to be a tilesheet with dimensions w,h.
 
 Colorize the textures with the given color.
 `<color>` is specified as a `ColorString`.
-`<ratio>` is an int ranging from 0 to 255 or the word "`alpha`".  If
+`<ratio>` is an int ranging from 0 to 255 or the word "`alpha`". If
 it is an int, then it specifies how far to interpolate between the
 colors where 0 is only the texture color and 255 is only `<color>`. If
 omitted, the alpha of `<color>` will be used as the ratio.  If it is
 the word "`alpha`", then each texture pixel will contain the RGB of
 `<color>` and the alpha of `<color>` multiplied by the alpha of the
 texture pixel.
 
+#### `[colorizehsl:<hue>:<saturation>:<lightness>`
+
+Colorize the texture to the given hue. All pixels will be converted
+to the specified hue while retaining their saturation and lightness, like
+"Colorize" in GIMP. Saturation and lightness can optionally be adjusted.
+
+`<hue>` should be from -180 to +180
+
+`<saturation>` and `<lightness>` are optional, and both from -100 to +100
```

Comment:
are these relative or absolute?

Reply:
the new text is using the degree symbol (°).

My editor thinks this file is unix utf-8 format, and I saw there's a PR/issue to switch it to a .md file, so it should be ok, but thought I'd call it out.